js/assessment.js

```diff
@@ -212,10 +212,10 @@ define([
                 for (var i = 0, l = assessments.length; i < l; i++) {
                     var assessmentState = assessments[i].getState();
 
-                    if (assessmentState.includeInTotalScore && !assessmentState.isPass) continue;
+                    if (assessmentState.includeInPageProgress && assessmentState.includeInTotalScore && !assessmentState.isPass) continue;
```

Comment:
Did you need to decrement the...
```js
var assessmentsTotal = assessments.length;
```
value according to the number of assessments that are included not just passed>

Reply:
Depends if you think that the code is clearer than incrementing assessmentsPassed when includeInPageProgress is false and assessmentState.isComplete is true.

Reply:
https://github.com/adaptlearning/adapt-contrib-assessment/blob/8ce40146a0081fee3e40c9bfa246b5ec45bdcb09/js/assessment.js#L209-L228

I means in the sense that the total will still pull through. So despite having chosen not to include the assessment in the plp, at the moment the code is excluding it from the passed count but not the assessment count. To exclude it from plp you'll need to exclude it from both the total count and the passed count. Not just the passed count.

https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/blob/master/js/completionCalculations.js#L40-L41